azure-storage-common/src/Common/Internal/Resources.php

```diff
@@ -46,6 +46,7 @@ class Resources
     const ACCOUNT_NAME_NAME = 'AccountName';
     const ACCOUNT_KEY_NAME = 'AccountKey';
     const SAS_TOKEN_NAME = 'SharedAccessSignature';
+    const USE_OAUTH = 'UseOAuth';
```

Comment:
`UseOAuth` seems more like a boolean instead of OAuth token value. Can we use "OAuthToken" or some friendly names?